src/components/tl/sharp/tl_sharp.h

```diff
@@ -62,6 +62,7 @@ UCC_CLASS_DECLARE(ucc_tl_sharp_lib_t, const ucc_base_lib_params_t *,
 typedef struct ucc_tl_sharp_oob_ctx {
     void           *ctx;
     ucc_oob_coll_t *oob;
+    ucc_subset_t    subset;
```

Comment:
is union will be better here? only oob or only subset is needed

Reply:
done